lms/djangoapps/certificates/queue.py

```diff
@@ -214,6 +214,18 @@ def add_cert(self, student, course_id, course=None, forced_grade=None, template_
         Returns the newly created certificate instance
         """
 
+        if not hasattr(course_id, 'ccx'):
```

Comment:
Why `hasattr` as opposed to an explicit check for `isinstance` against a CCX key?